client/src/components/NavHeader.tsx

```diff
@@ -1,18 +1,33 @@
 import { FC } from "react";
 import "../styles/NavHeader.scss"; // Importiamo il file SCSS personalizzato
-
+import { useNavigate } from "react-router-dom";
 interface NavHeaderProps {
   logout: () => void;
+  login: boolean;
 }
 
 const NavHeader: FC<NavHeaderProps> = (props): JSX.Element => {
+  const nav = useNavigate();
   return (
     <nav className="nav-header">
       <div className="nav-container">
-        <div className="nav-brand">Next Project</div>
-        <button className="btn-logout" onClick={props.logout}>
-          Logout
-        </button>
+        <div className="nav-left">
+          <img
+            src="/public/LOGO.png"
+            alt="Kiruna eXplorer Logo"
+            className="nav-logo"
+          ></img>
+          <div className="nav-brand">Kiruna eXplorer.</div>
+        </div>
+        {props.login ? (
+          <button className="btn-logout" onClick={props.logout}>
```

Comment:
Beware dashing create a single class named "btn-logout". You can assign two separate classes intead with className="btn logout" and if you want to select elements with both classes in CSS you just:

`.btn.logout { /* your rules */}`